domain/application/types.go

```diff
@@ -84,3 +84,15 @@ type UpsertUnitArg struct {
 	PasswordHash   *string
 	CloudContainer *CloudContainer
 }
+
+// These aliases are used to specify secret filter terms.
+type (
+	ApplicationSecretOwners []string
```

Comment:
What are these for?

Reply:
ah, left over from moving the logic to secret service